pkg/scheduler/framework/runtime/framework.go

```diff
@@ -1230,7 +1434,7 @@ func (f *frameworkImpl) RunPermitPlugins(ctx context.Context, state *framework.C
 		waitingPod := newWaitingPod(pod, pluginsWaitTime)
 		f.waitingPods.add(waitingPod)
 		msg := fmt.Sprintf("one or more plugins asked to wait and no plugin rejected pod %q", pod.Name)
-		klog.V(4).InfoS("One or more plugins asked to wait and no plugin rejected pod", "pod", klog.KObj(pod))
+		logger.V(4).Info("One or more plugins asked to wait and no plugin rejected pod")
```

Comment:
Why remove the `pod` info?

Reply:
Previous PR https://github.com/kubernetes/kubernetes/pull/111155 missed it.  I have added the `pod` info back.